packages/chain-adapters/src/cosmossdk/CosmosSdkBaseAdapter.ts

```diff
@@ -0,0 +1,221 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+// no-unused-vars is temporarily disabled until more functions are implemented
+import { CAIP2, caip2, caip19 } from '@shapeshiftoss/caip'
+import { AssetNamespace } from '@shapeshiftoss/caip/dist/caip19/caip19'
```

Comment:
We should export this from the caip package instead of drilling down to dist

Reply:
^^^ what big kev said